### DISCUSSION TITLE
Cancel or refund

### DISCUSSION THREAD

**Discussion by hohyun6858** [aclap-dev/video-downloadhelper#1312] [License & Purchase requests]
### Your question

Hello. I purchased the premium version of this product for Firefox for $28.50, but it was a mistake. 
I only use Chrome. Therefore, I would like to know if it's possible to cancel or refund my purchase.
 I intend to buy the Chrome version instead, as the Firefox version does not provide premium access on Chrome.
 My purchase email address is jhh6858@naver.com. 
I look forward to your reply. Thank you.

### Addon & CoApp version and details

version: 9.1.0.44
target: google
channel: stable
lang: ko
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"key":"31652cf282384168","mismatch":true,"other_browser":"Firefox","this_browser":"Chrome"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Whale/3.28.266.11 Safari/537.36

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
